src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/service/ExampleApiService.kt

```diff
@@ -4,25 +4,30 @@ import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.WebClientResponseException
 import reactor.core.publisher.Mono
+import java.time.LocalDateTime
 
+// This is an example of how to write a service calling out to another service.  In this case we have wired up the
+// template kotlin with itself so that the template doesn't depend on any other services.
 @Service
-class TemplateKotlinApiService(
+class ExampleApiService(
```

Comment:
decided to rename service to indicate that it was an example, rather than a template kotlin service that should be used for all service calls